receiver/iisreceiver/integration_test.go

```diff
@@ -10,9 +10,10 @@ import (
 	"testing"
 	"time"
 
+	"go.opentelemetry.io/collector/component"
```

Comment:
This seems irrelevant to your change?

Reply:
`tidy` made this change, should I remove it?